armi/reactor/tests/test_blocks.py

```diff
@@ -811,26 +811,58 @@ def test_replaceBlockWithBlock(self):
         self.assertEqual(3, len(block))
         self.assertEqual(block.p.height, refHeight)
 
-    def test_getWettedPerimeter(self):
-        cur = self.block.getWettedPerimeter()
-
+    def test_getWettedPerimeterCircularInnerDuct(self):
```

Comment:
This test does not seem to do what the name implies. It seems like just a standard HexBlock without an inner duct.